kubernetes/README.md

```diff
@@ -926,24 +931,30 @@ metadata:
   name: default
 spec:
   mtls:
-    ca:
-      builtin: {}
-    enabled: true
+    enabledBackend: ca-1
+    backends:
+    - name: ca-1
+      type: builtin
+  metrics:
+    enabledBackend: prometheus-1
+    backends:
+    - name: prometheus-1
+      type: prometheus
   logging:
+    defaultBackend: logstash
     backends:
     - name: logstash
-      format: |
-        {
-            "destination": "%KUMA_DESTINATION_SERVICE%",
-            "destinationAddress": "%UPSTREAM_HOST%",
-            "source": "%KUMA_SOURCE_SERVICE%",
-            "sourceAddress": "%KUMA_SOURCE_ADDRESS%",
-            "bytesReceived": "%BYTES_RECEIVED%",
-            "bytesSent": "%BYTES_SENT%"
-        }
-      tcp:
+      format: '{"start_time": "%START_TIME%", "source": "%KUMA_SOURCE_SERVICE%", "destination": "%KUMA_DESTINATION_SERVICE%", "source_address": "%KUMA_SOURCE_ADDRESS_WITHOUT_PORT%", "destination_address": "%UPSTREAM_HOST%", "duration_millis": "%DURATION%", "bytes_received": "%BYTES_RECEIVED%", "bytes_sent": "%BYTES_SENT%"}'
```

Comment:
Can we use the structured ordering here? It's hard to grasp what this line means like this.

Reply:
@nickolaev, I want to get this out to the community first so ill include that in the universal PR that ill open today, along with the rebased fault injection pr